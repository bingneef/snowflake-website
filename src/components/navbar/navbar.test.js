import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Navbar from "./navbar";

describe("Navbar", () => {
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Navbar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<Navbar />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
