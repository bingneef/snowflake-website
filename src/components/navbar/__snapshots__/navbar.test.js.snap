// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches the snapshot 1`] = `
<nav
  aria-label="main navigation"
  className="navbar navbar transparent"
  role="navigation"
>
  <a
    className="navItem isActive"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    className="navItem"
    onClick={[Function]}
  >
    Portfolio
  </a>
  <a
    className="navItem"
    onClick={[Function]}
  >
    About
  </a>
  <a
    className="navItem"
    href="https://www.github.com/bingneef"
    target="_blank"
  >
    <span
      className="icon"
    >
      <i
        className="fa fa-github"
      />
    </span>
  </a>
</nav>
`;
